packages/vats/src/vat-bank.js

```diff
@@ -404,6 +405,10 @@ const prepareBank = (
   const makeBalanceUpdater = prepareBalanceUpdater(zone);
   const makeBankPurseController = prepareBankPurseController(zone);
 
+  const addressDenomToPurse = zone.mapStore('addressDenomToPurse');
```

Comment:
```suggestion
  // Using a purseProvider singleton requires a single map with composite keys (which we emulate, since
  // we know both address and denom are JSONable).  If we decide to partition the provider and use
  // brandToVPurse directly, we'd need ephemera for each `makeBank` call.
  const addressDenomToPurse = zone.mapStore('addressDenomToPurse');
```